src/app/reducers/ToggleReducer/index.jsx

```diff
@@ -0,0 +1,19 @@
+// toggle actions
```

Comment:
whats this?

Reply:
So this is an action which can be dispatched based on some event, e.g a user interaction, a react lifecycle a fetch event etc.

This is following a common pattern used when implementing state management, where an interaction should not directly modify the state. Instead it dispatches an action with a data payload. This action then triggers the reducer and it is the reducer that actually handles the updating of state.

I have left this in as this action will be used when we allow feature toggles to be updated client side.